internal/resolution/variable_sources/requiredpackage/required_package.go

```diff
@@ -14,48 +14,49 @@ import (
 	"github.com/operator-framework/operator-controller/internal/resolution/variable_sources/util/sort"
 )
 
-type RequiredPackageVariable struct {
+type Variable struct {
```

Comment:
Linter complained about name stuttering: when exported `requiredpackage.RequiredPackageVariable` vs `requiredpackage.Variable.

Same for `RequiredPackageOption` vs `Option` and `RequiredPackageVariableSource` vs `VariableSource`